string objectName
float64 x
float64 y
float64 z
float64 oriented_x
float64 oriented_y
float64 oriented_z
float64 oriented_w
---
bool success
string objectName
float64 x
float64 y
float64 z
float64 oriented_x
float64 oriented_y
float64 oriented_z
float64 oriented_w
